parser/parser/cc/grammars/typedruby.ypp

```diff
@@ -1865,15 +1866,40 @@ lbrace_cmd_block_start:
                         &case_body->whens,
                         else_ ? else_->tok : nullptr,
 			                  else_ ? else_->nod : nullptr, $4);
+                      driver.rewind_if_dedented($1, $4);
+                    }
+                | kCASE expr_value opt_terms kEND
```

Comment:
Can we format this so that we have whitespace where `case_body` would go, so it's more obvious how this alternative differs from existing code?

```suggestion
                | kCASE expr_value opt_terms           kEND
```